Wikipedia/Code/MWKImage.h

```diff
@@ -46,6 +46,22 @@
 
 
 /**
+ * Width of the original image file. For example, if this image is a thumbnail, this is the width of the original image.
+ */
+@property (copy, nonatomic) NSNumber* originalFileWidth;
```

Comment:
@joewalsh are we using these new properties anywhere?

I didn't see them used except for when we inject them. Is there an intended place that you think we will be using this information?